.drone.yml

```diff
@@ -18,3 +18,28 @@ steps:
     CGO_ENABLED: 1
     GO111MODULE: on
     GOPROXY: https://proxy.golang.org
+
+- name: docker
+  image: plugins/docker
+  settings:
+    registry: quay.io
+    repo: quay.io/observatorium/thanos-receive-controller
+    password:
+      from_secret: quay_password
+    username:
+      from_secret: quay_username
+  when:
+    branch:
+    - master
```

Comment:
Does this mean we'll be publishing it on pushes to the `:master` tag? (I'm in favor of this, just trying to understand)

Reply:
Not quite, this means it will run this stage (docker build + push) only when there’s a push event on the master branch of the repo. We can specify the container tags separately

Reply:
So which tag is it pushed to?

Reply:
Right now it will use the default, which is `latest`

Reply:
Ok. Maybe we should change that in the future as kubernetes has some extra weird behavior with latest tags.